agent/bench-scripts/pbench-linpack

```diff
@@ -266,7 +282,8 @@ for thread in ${threads//,/ }; do
 
 		# Post-process the benchmark results after the tools so that the
 		# post-processing has that data available to it.
-		postprocess-iteration "${sample_dir}" "${benchmark_run_dir}"
+		[[ ${sample} -ne ${nr_samples} ]]; last=${?}
```

Comment:
Come to find out, Bash has a ternary operator!  So this can be either
```
let last=(sample == nr_samples ? 1 : 0)
```
or
```
(( last = (sample == nr_samples ? 1 : 0) ))
```
🙂